packages/zoe/src/cleanProposal.js

```diff
@@ -162,5 +162,5 @@ export const cleanProposal = (getAmountMath, proposal) => {
     );
   });
 
-  return harden({ want, give, exit });
+  return Data({ want, give, exit });
```

Comment:
If we know the data object can never be empty, we don't structly need to add `Data`, and it's probably better to omit it so the delta will be smaller later (when we strip Data out entirely). So I'd recommend against adding it here.

Reply:
Yes, I know. It's difficult to know a data object can never be empty in some cases though, and it rather easy to remove.